UPGRADE.md

```diff
@@ -22,12 +22,27 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 - added [Microservice Product Search Export](https://github.com/shopsys/microservice-product-search-export)
     - check changes in the `docker-compose.yml` template you used and replicate them, there is a new container `microservice-product-search-export`
     - `parameters.yml.dist` contains new parameter `microservice_product_search_export_url`
+- to use custom postgres configuration check changes in the `docker-compose.yml` templates and replicate them, there is a new volume for `postgres` container
+    - PR [Improve Postgres configuration to improve performance](https://github.com/shopsys/shopsys/pull/444)
+    - Move data from `project-base/var/postgres-data` into `project-base/var/postgres-data/pgdata`. Instructions for Ubuntu:
+        - `docker-compose down`
+        - `sudo su`
+        - `cd project-base/var/postgres-data/`
+        - trick to create directory `pgdata` with correct permissions
+            - `cp -rp base/ pgdata`
+            - `rm -fr pgdata/*`
+        - `shopt -s extglob dotglob`
+        - `mv !(pgdata) pgdata`
+        - `shopt -u dotglob`
+        - `exit`
+        - *update postgres container in `docker-compose.yml`*

```

Comment:
this is a little bit confusing - I have already updated my `docker-compose.yml` in the first upgrade step ("...check changes in `docker-compose.yml` templates and replicate them..."). So I do not need to do it again :slightly_smiling_face: